src/main/java/org/ibp/api/java/germplasm/GermplasmService.java

```diff
@@ -59,4 +63,9 @@
 
 	List<org.generationcp.middleware.api.attribute.AttributeDTO> getGermplasmAttributesByCodes(Set<String> codes);
 
+	List<AttributeDTO> getGermplasmAttributes();
+
+	List<Method> getAllBreedingMethods();
```

Comment:
We have BreedingMethodService, and there we have getBreedingMethods, that can be used if we need to filter depending on the program where the user is located at. If not, then move this function to the resource.

Reply:
I removed the getAllBreedingMethods.